config/canonical.json

```
@@ -30055,8 +30055,11 @@
   },
   "shop/supermarket|Hoogvliet": {
     "count": 70,
+    "countryCodes": ["nl"], 
     "tags": {
       "brand": "Hoogvliet",
+      "brand:wikidata": "Q1642179",
+      "brand:wikipedia": "nl:Hoogvliet_(supermarkt)",
```

Can you remove the `_` - everything else looks great 👍 
```suggestion
      "brand:wikipedia": "nl:Hoogvliet (supermarkt)",
```